# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1456781956 18000
#      Mon Feb 29 16:39:16 2016 -0500
# Node ID f20af4d085e95d6522de83d755d75f1f6f9c82d8
# Parent  1233bf2a03a593dcdcc9ca5dfd6b6c8e5172738c
create .oct files in build tree (bug #46260)

* libinterp/module.mk (ALL_LOCAL_TARGETS):
Add $(OCT_FILES) to the list.

diff --git a/libinterp/module.mk b/libinterp/module.mk
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -141,16 +141,18 @@ include libinterp/dldfcn/module.mk
 if AMCOND_ENABLE_DYNAMIC_LINKING
   OCT_FILES = $(DLDFCN_LIBS:.la=.oct)
   DLD_LIBOCTINTERP_LIBADD = libinterp/liboctinterp.la
 else
   OCT_FILES =
   DLD_LIBOCTINTERP_LIBADD =
 endif
 
+ALL_LOCAL_TARGETS += $(OCT_FILES)
+
 libinterp_liboctinterp_la_SOURCES = \
   libinterp/octave.cc \
   libinterp/version.cc \
   $(LIBINTERP_OPERATORS_SRC) \
   $(TEMPLATE_INST_SRC)
 
 nodist_libinterp_liboctinterp_la_SOURCES = \
   libinterp/corefcn/mxarray.h \
